### DISCUSSION TITLE
[Feature] "New terminal" button - customization

### DISCUSSION THREAD

**Discussion by ObserverFF** [Eugeny/tabby#6627] [General]
Hi 

I started using Tabby recently, and it is a great app. Many thanks to Eugene and all other contributors.
One thing that I noted (Tabby windows version) When I click on + to add "New terminal" tab it is opening by default CMD, it would be great if that can be customized in the settings so that we can select what profile "New terminal" button will open.

Regards,
ObserverFF

![2022-06-18 09_33_47-Window](https://user-images.githubusercontent.com/107758116/174440089-04bfc72c-276c-4bab-a8f4-f6b8afacbb8a.png)

**Comment by Eugeny** [comment]
Thanks! You can choose a different default profile in Settings - Profiles and Connections.

**Reply by ObserverFF** [reply]
Thanks Eugene, and sincere apologies for my blindness.
